jaxrs-genericresponse/src/main/java/com/cerner/beadledom/jaxrs/ResponseToStringWrapper.java

```diff
@@ -0,0 +1,26 @@
+package com.cerner.beadledom.jaxrs;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * Simple class to implement {@link Response} to String.
+ */
+public class ResponseToStringWrapper {
```

Comment:
This is more of a utility class than a wrapper. I was imagining a class that takes in `Response` to it's constructor and then implements `Object#toString()`.